src/app/connections/exceptions-log/mock-data/exceptions-log-spec-data.ts

```diff
@@ -0,0 +1,27 @@
+import { IException } from "../exceptions-log.interface";
+
+export const exceptionsJson = `
+[
+   
+    {
+        "gmcId": "1",
+        "errorMessage": "Error message",
+        "timestamp": "2023-10-18T14:45:47.337",
```

Comment:
Will these times work in future? (I'm guessing the "flush" method used just returns this no matter what day it is but just checking)

Reply:
Yeah, this uses the test data so should always pass.